docs/docs/building-an-ecommerce-site-with-shopify.md

```diff
@@ -193,3 +194,8 @@ exports.createPages = async ({ graphql, actions }) => {
   })
 }
 ```
+
+## Additional Resources
+
+- [Gatsby Shopify Starter](https://shopify-demo.gatsbyjs.com/)
+- [Gatsby Shopify Hello World](/starters/ohduran/gatsby-starter-hello-world-shopify/)
```

Comment:
Do we still want to promote this other starter? If it's not up to date with the latest Shopify plugin I'd recommend we take it out. (We can still leave it in the starter library, but we don't need to promote it alongside the 30k one.)

Reply:
I wasn't sure what to do with that one, but I'll take your recommendation here – makes a ton of sense! We can probably consider more resources if/when we pick up Shopify starter work again in the future
```suggestion
```